test/e2e/storage/testsuites/base.go

```diff
@@ -507,6 +507,10 @@ func getVolumeOpsFromMetricsForPlugin(ms metrics.Metrics, pluginName string) opC
 }
 
 func getVolumeOpCounts(c clientset.Interface, pluginName string) opCounts {
+	if !framework.ProviderIs("gce", "gke", "aws") {
```

Comment:
I'm not a fan of hard-coding provider information anywhere in the code base. I would prefer to see a change that makes it possible for a provider to override the list. Of course, this make the scope of the patch get larger but as we're moving to remove more cloud provider code from upstream kubernetes it's important that we stop hard-coding provider behavior.

Reply:
could you clarify what you mean by this?

btw. This issue is breaking running the tests on a local machine so its pretty important to get in. But I'm willing to understand what you mean and create an issue to work on later